.github/workflows/load-test.yml

```diff
@@ -78,6 +78,34 @@ jobs:
     - name: Install all dependencies and symlink for ep_etherpad-lite
       run: src/bin/installDeps.sh
 
-    # configures some settings and runs npm run test
+    # configures some settings and runs npm run test for 25 seconds with 50 authors
```

Comment:
This comment is likely to drift out of sync with the code. I'd say leave out the "for 25 seconds with 50 authors". If you want it to be more readable then use more verbose flags like `--duration=25 --users=50`.

Reply:
why would it drift out of sync?

Reply:
It's common for people to bump a number and forget to update the comment to match. Then that leaves people scratching their heads.